test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
- I tried your previous suggestion and it didn't work
- The `after` hook has an `fs.unlink` call, so I think the file should get cleaned up.
- We could even go so far as to add it to the <kbd>.gitignore</kbd> if it concerns you.